build.gradle

```diff
@@ -270,11 +270,6 @@ project(':core') {
     compile 'com.101tec:zkclient:0.7'
     compile 'com.yammer.metrics:metrics-core:2.2.0'
     compile 'net.sf.jopt-simple:jopt-simple:4.9'
-    if (baseScalaVersion == '2.11') {
-      compile 'org.scala-lang.modules:scala-xml_2.11:1.0.4'
-      compile 'org.scala-lang.modules:scala-parser-combinators_2.11:1.0.4'
```

Comment:
We use `scala.util.parsing.json.JSON`, which uses parser combinators, do the tests pass for 2.11 with this change?